File: KampIntro_Odevler/Gun_05_Odev_05/Gun_05_Odev_05/Abstract/BaseSaleManager.cs
using Gun_05_Odev_05.Entities;
using System;
using System.Collections.Generic;
using System.Text;

namespace Gun_05_Odev_05.Abstract
{
    public abstract class BaseSaleManager : ISaleService
    {
        public virtual void Add(Sale sale, Player player)
        {
            Console.WriteLine("Oyun satışı gerçekleşti : " + sale.SaleId + " " + sale.SaleName + " " + sale.SalePlayerId);
        }
    }
}

File: Java_React_Ödevler/Java_React_Ödev_Gün_04_Ödev_03_OyunSatis/Gun_04_Odev_03/Abstract/BaseSaleManager.java

public abstract class BaseSaleManager implements SaleService {
	
	public void add(Sale sale, Player player)
    {
		System.out.println("Oyun satýþý gerçekleþti : " + 
							sale.getId() 		  + " " + sale.getName() 		+ " " + sale.getPlayerId() + " " +
							player.getFirstName() + " " + player.getLastName());
    }

}
